docs/latex/gufi_administrator.tex

```diff
@@ -0,0 +1,28 @@
+\documentclass{article}
+
+
+
```

Comment:
lots of unnecessary empty lines

Reply:
There's no need to remove every empty line. There's just no need for 2 and 3 empty lines everywhere.